app/src/Hero.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { Carousel } from 'react-materialize';
+
+class Hero extends Component {
+	render() {
+		return (
+			<Carousel options={{ fullWidth: true }}>
+		  		<div className='red'>
```

Comment:
your div content is at the same tab level as their container

Reply:
ahh I just copy and pasted and didn't check it. I"ll update